src/clj/rems/css/styles.clj

```diff
@@ -71,12 +79,12 @@
 
 (defn- generate-media-queries []
   (list
-   (stylesheet/at-media {:max-width (u/px 480)}
+   (stylesheet/at-media {:max-width (:xs bootstrap-media-breakpoints)}
                         [(s/descendant :.rems-table.cart :tr)
```

Comment:
How about we consider some alternative notations for this part:
- `xs`
- `breakpoint-xs`
- `lt-xs`
- `lt-breakpoint-xs`

The last two would require putting `{:max-width ...}` and we could have `lt` and `gt` variants. Clojure allows `<` and `>` in the fn names so we could also have `breakpoint<xs` which would work for me.

I would not like to leave here any unnecessary stuff like repeating bootstrap or media words because I think they do not add value. In the breakpoint definition earlier one could mention that these come from bootstrap one way or another.

Reply:
It's just a typical abbreviation for less-than i.e. `<`.

Reply:
as I mentioned on flowdock, it can be tricky, can be confused with. larger than

Reply:
What do you think about my comments from flowdock? Can we stick to just bootstrap media breakpoints?

Reply:
`gt` is usually used in this context to mean greater-than. see e.g. &lt; &gt;